log_in.html

```diff
@@ -6,6 +6,15 @@
     <title>Log in</title>
 </head>
 <body>
-    
+    <img src="" alt="" class="backgroundImg">
+    <div class="container">
+        <div class="login-card">
+            <h1 class="title" id="title">Login page</h1>
+            <input type="text" class="nameText" id="name" placeholder="User Name"><br>
+            <input type="password" class="passwordText" id="password" placeholder="Password"><br>
+            <p class="passFaild" id="WrongPass">Please enter your correct password</p>
```

Comment:
```suggestion
            <p class="passFailed" id="wrongPass">Please enter your correct password</p>
```

Please change those typos and inconsistent naming convention formats.